apps/backend/src/main.ts

```diff
@@ -9,13 +9,13 @@ const UsersRouter = require('./app/users/user.controller').UsersRouter;
 const AuthRouter = require('./app/auth/auth.routes').AuthRouter;
 const InstitutionsRouter = require('./app/institution/institution.controller').InstitutionRouter;
 const EventsRouter = require('./app/events/event.controller').EventsRouter;
-const MailRouter = require('./app/services/mail/mail.routes').MailRouter;
+//const MailRouter = require('./app/services/mail/mail.routes').MailRouter;
```

Comment:
```suggestion
//const MailRouter = require('./app/services/mail/mail.routes').MailRouter;
```
Borrar ya que me habían quedado mal a mi del PR anterior

Reply:
Se eliminan los comentarios.  Se setea la versión de Typescript según sugerencia